### DISCUSSION TITLE
Scan clucometer

### DISCUSSION THREAD

**Discussion by hanbozdugan** [NightscoutFoundation/xDrip#2800] [General]
With the latest version of xDrip+ from 23.04.2023. I decided to connect the glucometer via bluetooth. I let xDrip+ scan for bluetooth devices but it hangs up and won't let me scan. This is clearly a bug in the latest updates. Before, xDrip+ scanned perfectly and didn't reset.

**Comment by jamorham** [comment]
Thank you for reporting this

**Comment by jamorham** [comment]
Please let me know if this is fixed by the April 24th Nightly (available now) ?

**Reply by hanbozdugan** [reply]
Great. I'm justifiably impressed with the quick intervention. A thousand thanks. Everything is OK

**Reply by jamorham** [reply]
thanks for testing the nightlies!
